CHANGELOG.md

```diff
@@ -1,3 +1,45 @@
+## 1.10.0-beta.1
+
+### ✨ Features
+
+* Add `mapboxgl.preloadWorkerPool()` and `mapboxgl.removePreloadedWorkerPool()` methods to allow developers to optimize load times for their maps ([#9391](https://github.com/mapbox/mapbox-gl-js/pull/9391))
+* Add `index-of` and `slice` expressions to search arrays and strings for the first occurrence of a specified value and return a section of the original array or string ([#9450](https://github.com/mapbox/mapbox-gl-js/pull/9450)) (h/t [lbutler](https://github.com/lbutler))
+* Correctly set RTL text plugin status if the plugin URL could not be loaded. This allows developers to add retry logic on network errors when loading the plugin ([#9489](https://github.com/mapbox/mapbox-gl-js/pull/9489))
+
+### 🍏 Improvements
```

Comment:
I like the practice we adopt on mobile of being flexible with the sub-heading names when a release features a particularly large number of changes centered on a specific category. For example, see how the iOS team handled the recent snapshotter refactor: https://github.com/mapbox/mapbox-gl-native-ios/releases/tag/ios-v5.8.0 

For this release, the changelog could be organized as:
```
### Features

### Gestures

### Other improvements
```

Reply:
Updated